src/app/[locale]/dashboard/account/page.tsx

```
@@ -0,0 +1,27 @@
+      <div className="container mx-auto flex gap-6 py-8 ">
+        <div className="w-1/4">
+          <h1>Placeholder...</h1>
+        </div>
+        <div className="w-3/4 p-4">
+          <h1 className="text-zinc-800 text-2xl font-semibold capitalize mb-9 dark:text-zinc-50">{t("account-settings")}</h1>
+          <ProfileForm profile= {profile} showChangePassword= {true} />
+        </div>
+      </div>
+    </div>
+  )
+}
```

ESLint

[error] Missing semicolon.

(semi)